### ISSUE TITLE
"ECMA-48 was last revised in ?1979?"

### ISSUE THREAD

**Issue by hackerb9** [wez/ecma48#3]
The README says that "ECMA-48 was last revised in 1979", but that doesn't sound right to me. The second edition was released in 1979, third and fourth in the 80s, and [fifth in 1991](https://www.ecma-international.org/wp-content/uploads/ECMA-48_5th_edition_june_1991.pdf).

You probably already knew that, but maybe consider updating the README so that folks like me don't stumble upon the page and get confused.

**Comment by Lieutenant-L-T-Smash** [comment]
Indeed, this project appears to be the HTML version of the _second edition_ of ECMA-48. The current edition is the fifth edition, which is downloadable as a searchable PDF from the ECMA website.

**Comment by PolyMeilex** [comment]
Yep, I also got bit by this when wondering why my paragraph links point to wrong values eg. CBT is `8.3.7` in ECMA 5th edition, and maps to `7.2.2` in this repo.

Kinda sad because I was hopping to use this as a linkable version of the spec.
